master/reference/networking-calico/floating-ips.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Floating IPs
+---
+
+networking-calico includes beta support for floating IPs.  Currently this
```

Comment:
I wonder if this is really "reference" material.  Feels more like a tutorial or guide to me.